jdk/src/share/classes/javax/net/ssl/SSLEngine.java

```diff
@@ -1350,8 +1352,9 @@ public String getHandshakeApplicationProtocol() {
      *         });
      * }</pre>
      *
-     * @apiNote
-     * This method should be called by TLS server applications before the TLS
+     * @apiNote This method is defined in Java SE 8 Maintenance Release 3.
+     *
```

Comment:
IMHO, this needs a separator here to separate the existing API from the addition.  Or maybe a second @apiNote?   It currently renders as this.

    API Note:
    This method is defined in Java SE 8 Maintenance Release 3. This method should be called by TLS server application...

Reply:
I suggest using <p> to start with a new paragraph.

Reply:
I'm guessing @mlchung meant `<p>`.  It didn't render correctly.

Reply:
Yes, that's what I meant.  I updated the formatting of my comment.

Reply:
FYI: a second `@apiNote` does not render correctly, suggest using <p> which I've verified does.

    API Note:
    This method is defined in Java SE 8 Maintenance Release 3.

    This method should be called by TLS server...